cms/settings/base.py

```diff
@@ -548,6 +555,16 @@
 # that let us use SMTP.
 # https://docs.djangoproject.com/en/2.1/topics/email/
 
+EMAIL_BACKEND = env.get("EMAIL_BACKEND", "django.core.mail.backends.console.EmailBackend")
+FROM_EMAIL = env.get("FROM_EMAIL", "example@mail.com")
+
+if EMAIL_BACKEND == "django_ses.SESBackend":
+    AWS_SES_REGION_NAME = env["AWS_REGION"]
+    AWS_SES_REGION_ENDPOINT = env["AWS_SES_ENDPOINT"]
```

Comment:
@RealOrangeOne is there anything we need to consider here (I am thinking about the IAM note from the other week)

Reply:
I think I'd recommend separating out the SES integration from adding notifications. Letting the boundary set in `send_mail` is fine.

The SES integration needs proper investigation to confirm it's setup correctly, and I think that should be kept separate from integrating this feature (or any feature, for that matter)

Reply:
There's already one from the platform side, so I think it's ok to cover that there: https://jira.ons.gov.uk/browse/DIS-2881